recipe/meta.yaml

```
@@ -1,43 +1,68 @@
 {% set name = "jsonlines" %}
-{% set version = "2.0.0" %}
+{% set version = "4.0.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6fdd03104c9a421a1ba587a121aaac743bf02d8f87fa9cdaa3b852249a241fe8
+  sha256: 0c6d2c09117550c089995247f605ae4cf77dd1533041d366351f6f298822ea74
 
 build:
-  noarch: python
-  number: 2
-  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  skip: true # [py<38]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - python
     - pip
+    - wheel
```

```suggestion
    - wheel
    - setuptools
```
Need `setuptools` see https://github.com/wbolster/jsonlines/blob/master/setup.py#L1